src/mgr/MgrStandby.cc

```diff
@@ -160,8 +161,25 @@ void MgrStandby::send_beacon()
   auto addr = available ? active_mgr->get_server_addr() : entity_addr_t();
   dout(10) << "sending beacon as gid " << monc.get_global_id()
 	   << " modules " << modules << dendl;
-
+  
+  const char *host_ip = NULL;
+  char addr_buf[129];
   map<string,string> metadata;
+  auto addr_standby = monc.get_my_addr();
+
+  switch(addr_standby.get_family()) {
+  case AF_INET:
+    host_ip = inet_ntop(AF_INET, &addr_standby.in4_addr().sin_addr, addr_buf,
+                        INET_ADDRSTRLEN);
+    break;
+  case AF_INET6:
+    host_ip = inet_ntop(AF_INET6, &addr_standby.in6_addr().sin6_addr, addr_buf,
+                        INET6_ADDRSTRLEN);
+    break;
+  default:
+    break;
+  };
+  metadata["addr"] = host_ip;
```

Comment:
@jcsp 
I assigned a value of “0.0.0.0” when "host_ip" is NULL, and log it,
I'm not sure if there is a better way.
thank you